db/pf-schema-X.Y.sql

```diff
@@ -1456,7 +1456,6 @@ CREATE TABLE `pki_certs` (
   `dns_names` varchar(255) DEFAULT NULL,
   `ip_addresses` varchar(255) DEFAULT NULL,
   `scep` BOOLEAN DEFAULT FALSE,
-  UNIQUE KEY `cn` (`cn`),
```

Comment:
I think we may need a new UNIQUE index on cn+profile_name or at least an index just to speed up the querying